content/news/2019-09-06.md

```diff
@@ -0,0 +1,25 @@
+---
+path: '/09-06-2019/spring-cloud-data-flow-2-3-0-m1-and-2-2-1-ga-released'
+title: 'Spring Cloud Data Flow 2.3.0.M1 and 2.2.1 GA Released'
```

Comment:
For older releases, it is probably better to just stick with GA release announcements. Anything on the current release line, however, we could highlight the latest milestones going forward.